tools/lsp/ui/api.slint

```diff
@@ -385,6 +404,57 @@ export global Api {
 
     ];
 
+    // ## Runtime properties
+
+    in-out property <[RuntimeComponent]> runtime-components: [
+        {
+            component-name: "Fruits",
```

Comment:
(🍍🍎🍓🍇 Typically something that can be replaced by json in the not so far future)

Reply:
I do not get what you want to tell me here, sorry :-) Use more emoji?